examples/classic/distributed/RockPaperScissor/play_rock_paper_scissor.qnt

```diff
@@ -0,0 +1,109 @@
+
+module rock_paper_scissor {
+
+    // Constants for player names
+    const PLAYER1 : str
+    const PLAYER2 : str
+
+    // Enum types for the possible moves and game status
+    type Move = Init | Rock | Paper | Scissor
+    type GameStatus = Started | Pending | Draw | Winner(str)
+
+    // Variables to hold the current state of each player's move and the game status
+    var p1State : Move
+    var p2State : Move
+    var game : GameStatus
```

Comment:
In general, modern languages don't have spaces between name and type, so I prefer not to add those
```suggestion
    var p1State: Move
    var p2State: Move
    var game: GameStatus
```